### PR TITLE
Odigos version v1.0.22

### PR DESCRIPTION
**Pull request by esara**
https://github.com/keyval-dev/odigos-charts/issues/18

### CODE REVIEW

File: charts/odigos/templates/autoscaler/deployment.yaml [hunk 1]
```diff
@@ -74,4 +57,8 @@ spec:
       serviceAccountName: odigos-autoscaler
       terminationGracePeriodSeconds: 10
       nodeSelector:
-        kubernetes.io/os: linux
\ No newline at end of file
+        kubernetes.io/os: linux
+{{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.imagePullSecrets | indent 6 }}
+{{- end }}
```
Comment (blumamir) [inline charts/odigos/templates/autoscaler/deployment.yaml:64]: I can't find this part in the [go cli code](https://github.com/keyval-dev/odigos/blob/f1c09476d025d8b13ffadea4532aa083694264a4/cli/cmd/resources/autoscaler.go#L355), but it looks like we should add it there as well. WDYT?

File: charts/odigos/templates/autoscaler/deployment.yaml [hunk 2]
```diff
@@ -1,58 +1,41 @@
...
-          - --upstream=http://127.0.0.1:8080/
-          - --logtostderr=true
-          - --v=0
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0
-        name: apis-rbac-proxy
-        ports:
-          - containerPort: 8443
-            name: https
-            protocol: TCP
       - args:
```
Comment (blumamir) [inline charts/odigos/templates/autoscaler/deployment.yaml:21]: I think we should also add the `imagePrefix` arg here if present in the values?
similar to what the CLI is doing [here](https://github.com/keyval-dev/odigos/blob/f1c09476d025d8b13ffadea4532aa083694264a4/cli/cmd/resources/autoscaler.go#L461)?

We can optionally address it in this PR or merge as is and add in the future

File: charts/odigos/templates/autoscaler/deployment.yaml [hunk 3]
```diff
@@ -74,4 +57,8 @@ spec:
       serviceAccountName: odigos-autoscaler
       terminationGracePeriodSeconds: 10
       nodeSelector:
-        kubernetes.io/os: linux
\ No newline at end of file
+        kubernetes.io/os: linux
```
Comment (blumamir) [inline charts/odigos/templates/autoscaler/deployment.yaml:60]: Seems this key is not present in the code anymore, we can remove it.

File: charts/odigos/templates/odiglet/daemonset.yaml [hunk 4]
```diff
@@ -1,38 +1,22 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
+  labels:
+    app.kubernetes.io/name: odiglet
```
Comment (blumamir) [inline charts/odigos/templates/odiglet/daemonset.yaml:5]: I think this label can be removed altogether from the daemonset object itself

**Review by blumamir — APPROVED** [review]

File: charts/odigos/templates/autoscaler/deployment.yaml (hunk 1 above)
Comment (esara) [reply charts/odigos/templates/autoscaler/deployment.yaml:64]: yes, probably - it was in the helm chart before, I just moved it around - probably makes sense to support this in the cli, too, if you also support custom registry locations

Comment (esara) [reply charts/odigos/templates/autoscaler/deployment.yaml:64]: this is just following k8s recommended labels
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

File: charts/odigos/templates/odiglet/daemonset.yaml (hunk 4 above)
Comment (esara) [reply charts/odigos/templates/odiglet/daemonset.yaml:5]: well, it does not really to hurt to have recommended labels, even if not needed for selectors

File: charts/odigos/templates/autoscaler/deployment.yaml [hunk 5]
```diff
@@ -1,58 +1,41 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  annotations:
-    odigos.io/skip: "true"
   labels:
-    app: odigos-autoscaler
+    app.kubernetes.io/name: odigos-autoscaler
```
[the remark this answers is not in the document]
Comment (esara) [reply charts/odigos/templates/autoscaler/deployment.yaml:5]: this is just following k8s recommended labels
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

File: charts/odigos/templates/autoscaler/deployment.yaml (hunk 2 above)
Comment (esara) [reply charts/odigos/templates/autoscaler/deployment.yaml:21]: added

**Pull request merged**